.github/workflows/seeder-tests.yml

```
@@ -0,0 +1,137 @@
+      - name: Run seeder tests
+        working-directory: apps/api
+        env:
+          DATABASE_URL: postgresql://postgres:postgres@localhost:5432/test_db
+          NODE_ENV: test
+        run: |
+          chmod +x scripts/test-seeders-ci.sh
+          ./scripts/test-seeders-ci.sh
+
+      - name: Upload test results
+        if: always()
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)